src/chi/utilities/position.scss

```diff
@@ -39,17 +39,17 @@
             position: #{$type};
           }
 
-          .-position-#{$size}--#{$type}-top {
-            position: #{$type};
-            top: 0;
+          .-position-#{$size}--#{$type}-bottom {
             left: 0;
+            position: #{$type};
             right: 0;
+            top: 0;
```

Comment:
@davidojedacl this class is not working beacuse of the top property, furthermore theer is no bottom position defined